Frameworks/UIKit.Xaml/Button.xaml.cpp

```diff
@@ -47,9 +45,19 @@ Button::Button() {
 
 // Accessor for our Layer content
 Image^ Button::LayerContent::get() {
-    // TODO: Add support for this.  Perhaps we return the same button's image, or we add another image within the canvas.  It should
-    // go just in front of the other button text/image, but behind any sublayers added by CoreAnimation via the SublayerCanvas property.
-    return nullptr;
+    if (!_content) {
+        Canvas^ rootCanvas = safe_cast<Canvas^>(GetTemplateChild(L"rootCanvas"));
```

Comment:
>Canvas^ rootCanvas = safe_cast<Canvas^>(GetTemplateChild(L"rootCanvas")); [](start = 8, length = 73)

we should probably cache this guy just like we do the others (in OnApplyTemplate)